lib/common.js

```diff
@@ -1,4 +1,10 @@
 module.exports = {
-  trackRoot: "https://track.customer.io/api/v1",
-  apiRoot: "https://api.customer.io/v1",
+  trackRoot: {
+    us: 'https://track.customer.io/api/v1',
+    eu: 'https://track-eu.customer.io/api/v1',
+  },
+  apiRoot: {
+    us: 'https://api.customer.io/v1',
+    eu: 'https://api-eu.customer.io/v1',
+  },
```

Comment:
We should probably add TypeScript support for this, or make these `us` and `eu` value constants since they are used in a few different places now. 🤔